cli/src/build/onboarding/android/ui/app.tsx

```
@@ -510,6 +532,84 @@ const AndroidOnboardingApp: FC<AppProps> = ({ appId, initialProgress, androidDir
+          const selected = await openServiceAccountJsonPicker()
+          if (cancelled)
+            return
+          if (!selected) {
+            // Cancelled — fall back to manual input. Reset the chooser screen
+            // so we don't loop back into picker mode immediately.
+            setSaJsonPathMode('manual')
+            setStep('sa-json-existing-path')
+            return
+          }
+          setServiceAccountJsonPath(selected)
+          await persist((p) => ({ ...p, serviceAccountJsonPath: selected }))
```

ESLint

[error] Unexpected parentheses around single function argument having a body with no curly braces.

(style/arrow-parens)